helm/eirini/templates/deployment.yaml

```diff
@@ -57,7 +57,7 @@ spec:
           readOnly: false
       containers:
       - name: opi
-        image: eirini/opi:{{ .Values.opi.image_tag }}
```

Comment:
We think it would be better (and easier for us) if we separate these into two values `opi.image_name` and `opi.image_tag`